src/app/design/styles/color/index.html

```diff
@@ -355,6 +355,156 @@
 	</div>
 		
 	<div>
+  <stache-page-anchor>
+    Containers
+  </stache-page-anchor>
+    <p>
+    Use container colors when categorizing blocks of content that will be mixed together, such as a calendar with different types of events. Container colors can be chosen specifically by using their variable name, or applied in a sequence.
+    </p>
+        
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="swatch" style="background-color: #bfecfb"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>Background color for categorized containers or for containers that have no categories.</div>
```

Comment:
For consistency with the existing descriptions on the page, delete "color." ... Same suggestion for the rest of the entries in this section.

This description confuses me. As written, it sounds like this is the the background for containers that include categories or for containers without categories, which sounds like this is the background for all containers, right? But I assume we are making a distinction between things that just isn't clear. At first I thought the problem was "categorized containers" and that it should be "categories within containers"? I **think** that might be what we mean, but I'm not sure. I think we are using "containers" to mean two different things here. The intro description makes it sound like containers ARE the categories, but this description makes it sound like containers contain the categories. So we seem to be using "containers" to mean both the categories within a container and the container itself. Either that or I'm just missing something. ... Either way, I have the same question/suggestion for the rest of the entries in this section.

Reply:
Maybe we get more clarity by labeling this set of colors "Categorizations" instead of "Containers". The explanation (and description of each row) probably gets simpler if we just start with that framing.